src/main/java/uk/gov/hmcts/ethos/replacement/docmosis/service/casetransfer/CaseTransferToEcmParams.java

```diff
@@ -0,0 +1,22 @@
+package uk.gov.hmcts.ethos.replacement.docmosis.service.casetransfer;
+
+import lombok.Builder;
+import lombok.Getter;
+
+import java.util.List;
+
+@Builder
+@Getter
+public class CaseTransferToEcmParams {
+    private String userToken;
+    private String caseTypeId;
+    private String jurisdiction;
+    private List<String> ethosCaseReferences;
+    private String newCaseTypeId;
+    private String positionType;
+    private String reason;
+    private String multipleReference;
```

Comment:
Does multipleReference serve any purpose? If not, can we get rid of it and any others that aren't used?

Reply:
Parameters removed